mitreattack/navlayers/generators/gen_helpers.py

```diff
@@ -1,6 +1,9 @@
-def remove_revoked(listing):
+mitre_attack_type_strings = ['mitre-attack', 'mitre-mobile-attack', 'mitre-ics-attack']
```

Comment:
Since this is a const [I believe it should be all caps](https://www.python.org/dev/peps/pep-0008/#constants)

Reply:
This is also not an "attack type". We usually use "attack type" to say "not STIX type", e.g "the ATT&CK type of the `intrusion-set` STIX type is `group`".

This is actually a variation on a [domain identifier](https://github.com/mitre-attack/attack-stix-data/blob/master/USAGE.md#domains).

Reply:
Updated the terminology.